scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -16,7 +16,7 @@
 # specific language governing permissions and limitations
 # under the License.
 # Might be empty
-AIRFLOW_COMMAND="${1:-}"
+readonly AIRFLOW_COMMAND="${1:-}"
```

Comment:
This is wrong too. In the "if" below the AIRFLOW_COMMAND might be overwritten so it cannot be readonly